setup/monitoring/stackdriver.md

```diff
@@ -108,11 +108,6 @@ Proceed to [install the dashboards](#installing-the-stackdriver-dashboards)
 
 ## Installing the Stackdriver dashboards
 
-*__Note__: The Stackdriver Dashboard API is currently whitelisted, so
-you need a registered STACKDRIVER_API key in order to use it. If you
-are so inclined, [contact your sales
-representative](https://cloud.google.com/contact/){:target="\_blank"}.
-
```

Comment:
This caveat is 3 years old and appears to no longer be true; as far as I can tell this is [GA now](https://cloud.google.com/monitoring/dashboards/api-dashboard).